app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -43,7 +39,8 @@ const Notes = () => {
   return (
     <Container>
       <Header
-        title="Notes"
+        menuBarToggle={function noRefCheck() {}}
```

Comment:
You have to go through the React best practices book before assigning me the PR again.

https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions#:~:text=Always%20prefer%20arrow%20functions

Reply:
Ok Yedhin. I went through the book again. I have corrected this as well.